includes/class-blocks.php

```diff
@@ -6,6 +6,11 @@
 
 class Blocks {
 	public static function init() {
+		// Short-circuit early if the site does not support blocks.
+		if ( ! \function_exists( 'register_block_type_from_metadata' ) ) {
```

Comment:
Should we add a function to check for block editor support? Something like `are_blocks_supported` and add also a filter?

Reply:
And maybe add the check to the main activitypub.php file, that would save a class init!

Reply:
All good ideas, I've just done that. 👍